### DISCUSSION TITLE
question: searchable with schemaless attribute

### DISCUSSION THREAD

**Discussion by ndarproj** [Power-Components/livewire-powergrid#792] [Q&A]
i'm using a package called https://github.com/spatie/laravel-schemaless-attributes in one of my columns.

i wanted to make it searchable but i'm encountering this
![image](https://user-images.githubusercontent.com/66521458/210706145-28307d00-0843-4851-b7ee-37ec39f2cdce.png)

addcolumns()
` return PowerGrid::eloquent()
            ->addColumn('meta_formatted', function ($model) {
                return $model->meta->toJson();
            });
    }`

columns()
`  Column::add()
                ->title('Meta')
                ->field('meta_formatted', 'meta')
                ->searchable()`

i dont think removing the cast for this package is possible. i'm looking for another way to be able to support search for this kind of columns

**Comment by luanfreitasdev** [comment]
This is not a powergrid bug, but we can assess the need to remove the cast in these cases. Could you share more details and how do I reproduce?
